extensions/monday/src/addItemToDefaultBoard.tsx

```diff
@@ -0,0 +1,76 @@
+import { Form, getPreferenceValues } from "@raycast/api";
+
+import { Board, Me } from "./lib/models";
+import { useState, useEffect } from "react";
+import { getBoardAndUser } from "./lib/api";
+import {
+  getCachedUser,
+  getCachedBoard,
+  cacheBoard,
+  cacheUser,
+} from "./lib/persistence";
+import AddItem from "./addItem";
+import { ErrorView } from "./lib/helpers";
+
+export default function AddItemToDefaultBoard() {
```

Comment:
I think calling the feature "Quick-add item" will be nicer and more obvious to people :) WDYT? 

```suggestion
export default function QuickAddItem() {
```

Reply:
good idea